modules/c/nrt/include/nrt/Utils.h

```diff
@@ -181,8 +181,17 @@ NRTAPI(char) nrt_Utils_cornersTypeAsCoordRep(nrt_CornersType type);
  * \param indexOne Index of first byte to be swapped
  * \param indexTwo Index of second byte to be swapped
  */
+/*
+ * Older versions of Visual Studio do not support `inline` for C
+ * Using `__inline` for Windows instead
+ */
+#ifdef WIN32
```

Comment:
You should actually be able to limit the ifdef area to just the `inline` keyword (in case we want to change the function signature in the future that way we don't have to change it in two spots).  Something like...

```
+/*
 + * Older versions of Visual Studio do not support `inline` for C
 + * Using `__inline` for Windows instead
 + */
NRTPRIV(void)
#ifdef WIN32
__inline
#else
inline
#endif
nrt_Utils_swap(nrt_Uint8* value, size_t indexOne, size_t indexTwo)
```